src/Modules/Data/Commands.Data/help/New-PowerBITable.md

```diff
@@ -0,0 +1,87 @@
+---
+external help file: Microsoft.PowerBI.Commands.Data.dll-Help.xml
+Module Name: MicrosoftPowerBIMgmt.Data
+online version:
+schema: 2.0.0
+---
+
+# New-PowerBITable
+
+## SYNOPSIS
+Creates New PowerBI table object.
+
+## SYNTAX
+
+```
+New-PowerBITable -Name <String> [-Columns <Column[]>] [<CommonParameters>]
+```
+
+## DESCRIPTION
+The New-PowerBITable cmdlet lets you create New PowerBI table object.
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\>$col1 = New-PowerBIColumn -Name ID -DataType Int64
+PS C:\>$col2 = New-PowerBIColumn -Name Data -DataType String
+PS C:\>$table1 = New-PowerBITable -Name SampleTable1 -Columns $col1,$col2
+PS C:\>
+PS C:\>$col3 = New-PowerBIColumn -Name ID -DataType Int64
+PS C:\>$col4 = New-PowerBIColumn -Name Date -DataType DateTime
+PS C:\>$col5 = New-PowerBIColumn -Name Detail -DataType String
+PS C:\>$col6 = New-PowerBIColumn -Name Result -DataType Double
+PS C:\>$table2 = New-PowerBITable -Name SampleTable2 -Columns $col3,$col4,$col5,$col6
+PS C:\>
+PS C:\>$dataset = New-PowerBIDataSet -Name SampleDataSet -Tables $table1,$table2
+PS C:\>
+PS C:\>Add-PowerBIDataSet -DataSet $dataset
+```
+
+This example instantiate a table with two columns and another table with four columns, and instantiate a dataset.
+Then, it creates the dataset in PowerBI.
+
+## PARAMETERS
+
+### -Columns
+An array of Column objects.
```

Comment:
lowercase on column, repeat for other instances for parameter names

Reply:
Done.